ajax/libs/focusable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "focusable",
+  "filename": "focus-element-overlay.min.js",
+  "version": "1.0.0",
+  "description": "Set focus on DOM element adding overlay to the rest of the page",
+  "homepage": "http://zzarcon.github.io/focusable/",
+  "keywords": [
+    "js",
+    "focusable"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/zzarcon/focusable.git"
+  },
+  "author": "zzarcon",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/zzarcon/focusable.git",
+    "fileMap": [
+      {
+        "basePath": "app",
+        "files": [
+          "focus-element-overlay.js",
```

Comment:
Let's use `**/*` to fetch all files under `app` recursively. For more details, you can refer https://github.com/isaacs/node-glob
(Let reviewers know: `app` plays the similar role to `dist` in this library)